File: ride-service/src/main/java/com/example/ride_service/model/dto/DriverResponseDto.java
package com.example.ride_service.model.dto;

import lombok.Builder;

import java.util.UUID;

@Builder
public record DriverResponseDto(
        UUID driverId,
        String name,
        String carBrand,
        String carModel,
        String carLicensePlate,
        String carColor) {
}

File: driver-service/src/main/kotlin/com/example/driver_service/model/dto/DriverResponseDto.kt
package com.example.driver_service.model.dto

import com.example.driver_service.model.enums.Gender
import java.math.BigDecimal
import java.util.UUID

data class DriverResponseDto(
    val id: UUID,
    val name: String,
    val email: String,
    val phoneNumber: String,
    val rating: BigDecimal?,
    val gender: Gender,
    val carId: UUID?,
)
